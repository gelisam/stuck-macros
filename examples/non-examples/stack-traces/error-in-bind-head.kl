#lang "prelude.kl"

(example (>>= (pure (error 'e)) (lambda (x) (pure x))))