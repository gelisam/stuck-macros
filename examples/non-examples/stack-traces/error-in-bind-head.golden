Error at phase p0:
    error-in-bind-head.kl:3.29-3.30: e
    stack trace:
    ---- in pure macro
    ---- in bind macro head
    Î»x. pure x
    ---- Halt
